docs/install/collecting-metrics/README.md

```diff
@@ -0,0 +1,101 @@
+This document describes how to set up the
+[OpenTelemetry Collector](https://opentelemetry.io/docs/collector/about/) to
+receive metrics from the Knative infrastructure components and distribute them
+to Prometheus. [OpenTelemetry](https://opentelemetry.io/) is a CNCF project to
+provide consistent interfaces for instrumenting applications and platforms to
+provide telemetry information for developers and operators. OpenTelemetry allows
+Knative to build provider-agnostic instrumentation into the platform, so that
+it's easy to export logs to multiple monitoring services without needing to
+rebuild or reconfigure the Knative binaries.
+
+## Setting up the collector
+
+The collector provides a long-lived location where various Knative components
+can push metrics (and eventually traces) to be retained and collected by a
+monitoring service. For this example, we'll configure a single collector
+instance using the
+[opentelemetry-operator](https://github.com/open-telemetry/opentelemetry-operator).
+
+![Diagram of components reporting to collector, which is scrapde by Prometheus](./system-diagram.svg)
+
+<!-- yuml.me UML rendering of:
+[queue-proxy1]->[Collector]
+[queue-proxy2]->[Collector]
+[autoscaler]->[Collector]
+[Collector]<-scrape[Prometheus]
+-->
+
+1. The first step is to install the operator:
+
+   ```shell
+   kubectl apply --filename https://github.com/open-telemetry/opentelemetry-operator/releases/latest/download/opentelemetry-operator.yaml
```

Comment:
When I run the command I get two warnings, I'm guessing there are optional CRs in the release yaml. harmless.
```
nable to recognize "https://github.com/open-telemetry/opentelemetry-operator/releases/latest/download/opentelemetry-operator.yaml": no matches for kind "Certificate" in version "cert-manager.io/v1alpha2"
unable to recognize "https://github.com/open-telemetry/opentelemetry-operator/releases/latest/download/opentelemetry-operator.yaml": no matches for kind "Issuer" in version "cert-manager.io/v1alpha2"
```
Should we mention something or not? I'm ok either way.

Reply:
We should add `kubectl wait` here if not on the next step to create a collector CR it would fail
```
Error from server (InternalError): error when creating "collector.yaml": Internal error occurred: failed calling webhook "mopentelemetrycollector.kb.io": Post "https://opentelemetry-operator-webhook-service.opentelemetry-operator-system.svc:443/mutate-opentelemetry-io-v1alpha1-opentelemetrycollector?timeout=30s": dial tcp 10.96.101.77:443: connect: connection refused
```
The operator pod was still creating
```
kubectl get pods -n opentelemetry-operator-system
NAME                                                         READY   STATUS              RESTARTS   AGE
opentelemetry-operator-controller-manager-6c7b945dcf-vqzvv   0/2     ContainerCreating   0          7m46s
```

Reply:
operator is not installing for me for some reason, I'm using kind
```
Unable to attach or mount volumes: unmounted volumes=[cert],
```
I will reach out on Slack @evankanderson

Reply:
I forgot to copy the instruction to install cert-manager, which I already had on the cluster for Knative.

I'll also see if there are any plans to avoid having a hard dependency on cert-manager.